### PR TITLE
🆕 Compatibility with Python 3.7

### PR DESCRIPTION
**Pull request by rowanseymour**
3.6 is still the official / tested Python but handy to get things working on 3.7 since that's what you'll get if you do a `brew update`

### CODE REVIEW

File: temba/ussd/models.py [hunk 1]
```diff
@@ -105,7 +105,7 @@ def handle_incoming(
         content=None,
         starcode=None,
         org=None,
-        async=True,
+        do_async=True,
```
Comment (rowanseymour) [inline temba/ussd/models.py]: `async` is a reserved word now

File: pip-freeze.txt [hunk 2]
```diff
@@ -66,7 +66,7 @@ isoweek==1.3.3
 jdcal==1.4                # via openpyxl
 jmespath==0.9.3           # via boto3, botocore
 jsondiff==1.1.1
-kombu==4.0.2              # via celery
+kombu==4.2.1              # via celery
```
Comment (dodobas) [inline pip-freeze.txt]: why are version of these modules also bumped if their 'parents' have not been?

Comment (rowanseymour) [reply pip-freeze.txt]: was going for minimal changes... but guess we could bump celery to latest 4.x

File: pip-freeze.txt [hunk 3]
```diff
@@ -102,7 +102,7 @@ python-intercom==3.1.0
 python-magic==0.4.15
 python-telegram-bot==6.1.0
 pytz==2017.2
-pyyaml==3.12              # via uservoice
+pyyaml==3.13              # via uservoice
```
Comment (norkans7) [inline pip-freeze.txt]: we no longer need uservoice

Comment (rowanseymour) [reply pip-freeze.txt]: Ha oh yeah forgot about that!

**Pull request merged**